data/profiles/default.yaml

```diff
@@ -11,6 +11,7 @@ spec:
     components:
       pilot:
         common:
+          enabled: true
```

Comment:
all the component level enabled settings are redundant, since the feature is enabled. here and below.

Reply:
I think adding per component enablement value makes it more clear to understand. Also when doing reverse translation from values.yaml to iscp, each component enablement value is added, so it makes easier and less ambiguous during comparison